### ISSUE TITLE
Clearer feedback if the plugin is working or not (Chrome)

### ISSUE THREAD

**Issue by seriema** [livereload/livereload-extensions#30]
I've only checked the Chrome icons but they're all the same. There's little (and unclear) difference if Livereload is active or not. It would be nice with a clearer visual queue.

Active: ![](https://raw.githubusercontent.com/livereload/livereload-extensions/master/Chrome/LiveReload/IconActive.png)
Disabled: ![](https://raw.githubusercontent.com/livereload/livereload-extensions/master/Chrome/LiveReload/IconDisabled.png)

Especially now that I've installed it on a new computer. I press the icon but I get no feedback. LR doesn't seem to work, but I have no idea why. A red icon or something when it's not working would be nice. Especially nice if it would output some kind of feedback in text somewhere (console maybe? popover?).

Maybe I'm missing something?

**Comment by tchalvak** [comment]
Yes please, a color change when active would also be great.  I'm trying to get the initial setup working, and the server-side is requesting handshakes and connecting, but the client side... ...shows nothing.

**Comment by stevesobol** [comment]
Great? Necessary. It's impossible to tell when the plugin is connected. In Vivaldi (Chromium-powered browser), and probably also in Chrome and the other major browsers, the little dot at the center of the icon is IMPOSSIBLE to see.

I see this issue has been open for 5-1/2 years, so I don't hold forth much hope of this getting done anytime soon, and unfortunately, right now I don't have time to fork the repo, make the change myself and send in a pull request... but this is something that really needs to be done.
